app/views/subscriptions/index.html.haml

```diff
@@ -0,0 +1,15 @@
+- content_for :page_title do
+  = t('.title')
+
+- content_for :content do
+  #container.container-fluid
```

Comment:
Classes should be listed before IDs (.container-fluid should precede #container)